coreneuron/nrniv/nrn_setup.cpp

```diff
@@ -1632,6 +1646,7 @@ for (int i=0; i < nt.end; ++i) {
         }
         nt._vecplay[i] = new VecPlayContinuous(ml->data + ix, yvec, tvec, NULL, nt.id);
     }
+    F.close();
```

Comment:
why we need to close it here? If I understood correctly, it's taken care by nrn_setup.h:

```c
file_reader_w[i].close();
```